public/get-results.js

```diff
@@ -0,0 +1,34 @@
+/* global: location, sessionStorage */
+
+(function resultsPage () {
```

Comment:
can you wrap the whole file in an iife, to stop your function names polluting the global scope